Dockerfile

```diff
@@ -6,7 +6,7 @@
 
 # Want to help us make this template better? Share your feedback here: https://forms.gle/ybq9Krt8jtBL3iCk7
 
-ARG PYTHON_VERSION=3.9.16
+ARG PYTHON_VERSION=3.9.19
```

Comment:
Would be nice to ratchet up to a newer version, but I won't block on this cause I am not some sort of jerk.

Reply:
Until https://github.com/oscal-compass/compliance-trestle/pull/1533 is merged and released, 3.9 is the most recent supported version.